sdk/schema/credential_type.go

```diff
@@ -33,6 +33,9 @@ type CredentialField struct {
 	// The name of the field, e.g. "Token", "Password", or "Username".
 	Name sdk.FieldName
 
+	// Other names this field can be recognized after. The order in which these names are specified matters towards which 1Password item field will be selected.
+	AlternativeNames []sdk.FieldName
```

Comment:
Maybe this should be `[]string` to give us the freedom to potentially clean out the `fieldname` package without being restricted by deprecated values.
```suggestion
	AlternativeNames []string
```

Reply:
Makes sense! Thanks